components/StarSearch/ChatAvatar.tsx

```diff
@@ -5,7 +5,7 @@ type Author = "You" | "StarSearch" | "Guest";
 
 interface ChatAvatarProps {
   author: Author;
-  userId?: number;
+  userId?: number | null;
```

Comment:
This change was made because we're getting the user ID client-side now and `useSupabaseAuth` can return null for a `userId`.